app/controllers/concerns/foreman/controller/puppet/hosts_controller_extensions.rb

```diff
@@ -1,184 +0,0 @@
-module Foreman::Controller::Puppet::HostsControllerExtensions
```

Comment:
Shouldn't this be part of the UI drop?
And we should drop it from https://github.com/theforeman/foreman/blob/324bafdb625b06fe939530ef77534170be855447/app/controllers/hosts_controller.rb#L10

Reply:
This is not resolved. This (`Foreman::Controller::Puppet::HostsControllerExtensions`) should be part of the UI drop, as it is UI controller extension, right?